packages/app-store/sylaps/_metadata.ts

```diff
@@ -0,0 +1,34 @@
+import type { AppMeta } from "@calcom/types/App";
+
+import _package from "./package.json";
+
+export const metadata = {
+  name: "Sylaps",
+  description: _package.description,
+  installed: true,
+  type: "sylaps_video",
+  imageSrc: "/api/app-store/sylaps/icon.svg",
+  variant: "conferencing",
+  categories: ["video"],
+  logo: "/api/app-store/sylaps/icon.svg",
+  publisher: "Cal.com",
```

Comment:
```suggestion
  publisher: "hthetiot",
```

Reply:
OR "Sylaps" if you belong to them

Reply:
Yes I'm Sylaps founder.
Updated to published to Sylaps Inc.
Thx for review.

Reply:
rest looks great! will try today. just made a sylaps account to tet